.dockerignore

```diff
@@ -5,7 +5,8 @@ static/*
 !static/images/*
 deploy/*
 !deploy/docker/seqr/*
-hail_search/*
+!deploy/docker/hail_search/*
+hail_search/fixtures/*
```

Comment:
one thing to note is this is including the hail search code in the main seqr docker image, which it previously wasn't. Not sure its a huge deal, but I liked having them be totally distinct

Reply:
What would be the list of directories that we'd want in the `seqr` Dockerfile?

```
admin
matchmaker
panelapp
reference_data
seqr
static
ui
```?  

I think being explicit is probably the best option at this point.

Reply:
this looks right to me